config/mainnet.go

```diff
@@ -0,0 +1,105 @@
+package config
+
+import (
+	"math/big"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/activation"
+	"github.com/spacemeshos/go-spacemesh/api/grpcserver"
+	"github.com/spacemeshos/go-spacemesh/beacon"
+	"github.com/spacemeshos/go-spacemesh/bootstrap"
+	"github.com/spacemeshos/go-spacemesh/checkpoint"
+	"github.com/spacemeshos/go-spacemesh/fetch"
+	hareConfig "github.com/spacemeshos/go-spacemesh/hare/config"
+	eligConfig "github.com/spacemeshos/go-spacemesh/hare/eligibility/config"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+	"github.com/spacemeshos/go-spacemesh/syncer"
+	timeConfig "github.com/spacemeshos/go-spacemesh/timesync/config"
+	"github.com/spacemeshos/go-spacemesh/tortoise"
+)
+
+func MainnetConfig() Config {
+	return Config{
+		BaseConfig: BaseConfig{
+			DataDirParent:       defaultDataDir,
+			FileLock:            filepath.Join(os.TempDir(), "spacemesh.lock"),
+			MetricsPort:         1010,
+			MetricsPushPeriod:   60,
+			DatabaseConnections: 16,
+			NetworkHRP:          "sm",
+
+			LayerDuration:  5 * time.Minute,
+			LayersPerEpoch: 4032,
+
+			TxsPerProposal: 700,       // TODO
+			BlockGasLimit:  100107000, // 3000 of spends
+
+			OptFilterThreshold: 90,        // TODO
+			TickSize:           1_000_000, // TODO
```

Comment:
80000 per second
that is 93312000000 ticks per epoch `(epoch_length * 60 * 60 - cycle_gap_seconds) * poet_ticks_per_second`

`epoch_length` 14 * 24 hours.
`cycle_gap_seconds` 43200

To normalize to 10k we need to make `tick_height = 9331200`


The poets are making 77-78k consistently. That leaves some buffer for future.